data/campaigns/fri05.wmf/scripting/init_p1.lua

```diff
@@ -0,0 +1,2975 @@
+          wares=             {
+            },
+        },
+      [103] =         {
+          x= 93,
+          y= 179,
+          wares=             {
+            },
+        },
+      [104] =         {
+          x= 91,
+          y= 184,
+          wares=             {
+            },
+        },
+      [105] =         {
+          x= 89,
+          y= 182,
+          wares=             {
+            },
+        },
+      [106] =         {
+          x= 87,
+          y= 181,
+          wares=             {
+            },
+        },
+      [107] =         {
+          x= 92,
+          y= 186,
+          wares=             {
+            },
+        },
+      [108] =         {
+          x= 91,
+          y= 188,
+          wares=             {
+            },
+        },
+      [109] =         {
+          x= 93,
+          y= 185,
+          wares=             {
+            },
+        },
+      [110] =         {
+          x= 91,
+          y= 190,
+          wares=             {
+            },
+        },
+      [111] =         {
+          x= 91,
+          y= 192,
+          wares=             {
+            },
+        },
+      [112] =         {
+          x= 89,
+          y= 189,
+          wares=             {
+            },
+        },
+      [113] =         {
+          x= 88,
+          y= 188,
+          wares=             {
+            },
+        },
+      [114] =         {
+          x= 91,
+          y= 194,
+          wares=             {
+            },
+        },
+      [115] =         {
+          x= 93,
+          y= 194,
+          wares=             {
+            },
+        },
+      [116] =         {
+          x= 92,
+          y= 197,
+          wares=             {
+            },
+        },
+      [117] =         {
+          x= 89,
+          y= 195,
+          wares=             {
+            },
+        },
+      [118] =         {
+          x= 91,
+          y= 199,
+          wares=             {
+            },
+        },
+      [119] =         {
+          x= 93,
+          y= 194,
+          wares=             {
+            },
+        },
+      [120] =         {
+          x= 90,
+          y= 200,
+          wares=             {
+            },
+        },
+      [121] =         {
+          x= 88,
+          y= 201,
+          wares=             {
+            },
+        },
+      [122] =         {
+          x= 89,
+          y= 133,
+          wares=             {
+            },
+        },
+      [123] =         {
+          x= 80,
+          y= 169,
+          wares=             {
+            },
+        },
+      [124] =         {
+          x= 83,
+          y= 173,
+          wares=             {
+            },
+        },
+    },
+}
+
+function place_flags(eco)
+
+   for i = 1, #eco.flags do
+      local flag = p1:place_flag(map:get_field(eco.flags[i].x, eco.flags[i].y),true)
+      for ware,count in pairs(eco.flags[i].wares) do
+         flag:set_wares(ware,count)
```

Comment:
this does not work for flags well. we need to set the table returned by the dump script (this is the error in the atl02 scenario as well see #5603)